webrender_traits/src/types.rs

```diff
@@ -371,10 +372,13 @@ pub enum MixBlendMode {
 pub type NativeFontHandle = CGFont;
 
 /// Native fonts are not used on Linux; all fonts are raw.
-#[cfg(not(target_os = "macos"))]
-#[cfg_attr(not(target_os = "macos"), derive(Clone, Serialize, Deserialize))]
+#[cfg(all(unix, not(target_os="macos")))]
```

Comment:
(@nox - @glennw left out a second added line which was the cfg_attr with the same conditional as the cfg)

Yeah, this is definitely a serde bug.  In the generated `types.rs`, this is expanding out to:
```
#[cfg(target_os = "macos")]
pub type NativeFontHandle = CGFont;
#[allow(non_upper_case_globals, unused_attributes, unused_qualifications)]
#[cfg(all(unix, not(target_os = "macos")))]
const _IMPL_DESERIALIZE_FOR_NativeFontHandle: () =
...
#[cfg(target_os = "macos")]
pub type NativeFontHandle = CGFont;
#[allow(non_upper_case_globals, unused_attributes, unused_qualifications)]
#[cfg(all(unix, not(target_os = "macos")))]
const _IMPL_DESERIALIZE_FOR_NativeFontHandle: () =
    {
...
    };
/// Native fonts are not used on Linux; all fonts are raw.
#[cfg(all(unix, not(target_os = "macos")))]
pub struct NativeFontHandle;
```
Note that all mention of `derive(Clone)` was lost.  Even if I add a second `cfg_attr` with just `derive(Clone)` it gets eaten as well.  Changing the conditional does not change the behaviour.  Forcing serde to 0.8.16 doesn't help.  Hrm.